sinatra/sinatra_app.rb

```diff
@@ -78,14 +78,18 @@ def authorized?
 
 post '/api/templates/:id/delete' do
   protected!
+  content_type :json
 
   id = params[:id].to_i
   dataset = database[:templates]
   dataset.filter(id: id).delete
+
+  halt 200, { id: id }.to_json
```

Comment:
Why is this a halt, specifically? And not otherwise setting the response code/body?